packages/EasyCore/src/Bridge/Symfony/Serializer/TrimStringsDenormalizer.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Bridge\Symfony\Serializer;
+
+use EonX\EasyCore\Helpers\StringsTrimmerInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+/**
+ * @SuppressWarnings(PHPMD.UnusedFormalParameter) Method signatures are defined by parent
+ */
+final class TrimStringsDenormalizer implements DenormalizerInterface
+{
+    /**
+     * @var \EonX\EasyCore\Helpers\StringsTrimmerInterface
+     */
+    private $trimmer;
+
+    /**
+     * @var \Symfony\Component\Serializer\Normalizer\DenormalizerInterface
+     */
+    private $decorated;
+
+    /**
+     * A list of array keys whose values will be ignored during processing.
+     * @see \EonX\EasyCore\Tests\Helpers\RecursiveStringsTrimmerTest for example
+     *
+     * @var string[]
+     */
+    private $except;
+
+    /**
+     * @param string[] $except
+     */
+    public function __construct(
+        DenormalizerInterface $decorated,
+        StringsTrimmerInterface $trimmer,
+        array $except = []
```

Comment:
Same here please use nullable for optional parameters.

Reply:
Done.